pytensor/link/numba/dispatch/elemwise.py

```diff
@@ -996,15 +1066,15 @@ def numba_funcify_MaxAndArgmax(op, node, **kwargs):
     if x_ndim == 0:
 
         @numba_basic.numba_njit(inline="always")
-        def maxandargmax(x):
-            return x, 0
+        def max(x):
+            return x
 
     else:
         axes = tuple(int(ax) for ax in axis)
```

Comment:
This causes problem when axis is ```None```.

Reply:
You'll need to convert into axis = tuple(range(x_ndim)). I assume this is only a problem for Argmax? I think the conversion is already done for Max by default (as is for all CAReduce)?

We can do the same conversion for Argmax. Are we always creating Argmax for the user in pt.argmax? If so we can do the conversion there already

Reply:
Yeh this code was redundant. ```numba_funcify_CAReduce``` already handles it effectively. ```Argmax``` behaves as expected already.